x-pack/plugins/fleet/server/routes/schema/errors.ts

```diff
@@ -10,8 +10,8 @@ import { schema } from '@kbn/config-schema';
 export const genericErrorResponse = () =>
   schema.object(
     {
-      statusCode: schema.number(),
-      error: schema.string(),
+      statusCode: schema.maybe(schema.number()),
```

Comment:
it seems in some cases `statusCode` is missing from the response, so making it optional
![image](https://github.com/user-attachments/assets/dc4d18dd-1eda-4987-a9bc-a4a953384427)